### DISCUSSION TITLE
Homologação de Participante Indireto

### DISCUSSION THREAD

**Discussion by viniciusrs95** [bacen/pix-api#439] [Q&A]
Pessoal, me tirem uma dúvida

Se estou oferecendo um serviço apenas de geração do qrCode dinâmico, geração de chaves aleatórias e envio de pagamento com informação de dados manuais. Trabalhando como participante indireto, não temos aplicativos, vamos oferecer API apenas. 
Nossos clientes não vão ter acesso à chave também, pois ela será usada apenas para o recebimento.

O Participante direto, pode querer exigir homologação de todas as funcionalidades do DICT mesmo que não vamos oferecer isso para os clientes (pessoas jurídicas) ? Por exemplo, eles querem que eu faça reivindicação de chaves, sendo que não vamos ter isso na operação. 

Pelo documento do Bacen um participante indireto não precisa passar pelo "Testes formais de homologação" apenas que o direto certifique que o indireto consiga operar.

"Os testes de homologação de que trata o caput deverão ser definidos pelo liquidante no SPI, de forma que ele seja
capaz de declarar a aptidão operacional do participante indireto."

Vocês acham correto o participante direto querer homologar um indireto com funcionalidades que não fazem sentido?

**Comment by rubenskuhl** [comment]
Pela sua descrição, você não poderia ser participante nem direto nem indireto. Me parece correto sim o que o participante direto está exigindo, e que ele se meteria em encrenca se te deixasse entrar.

**Reply by joelemanoel** [reply]
Além disso não vejo nenhuma restrição imposta pelo BACEN que proiba tal prática, além de acrescentar segurança a operação pela participante direta.

**Reply by viniciusrs95** [reply]
Se eu estou recebendo e liquidando PIX para vários clientes. Eu não deveria ser PSP? Não entendi, qual é o motivo dessa afirmação?

**Reply by ninrod** [reply]
@viniciusrs95 , recomendo expor sua dúvida ao DECEM em pix at bcb.gov.br

**Reply by viniciusrs95** [reply]
ok, obrigado pessoal
